src/extensions/scratch3_ev3/index.js

```diff
@@ -683,17 +679,18 @@ class EV3 {
             // GET DEVICE LIST
             byteCommands[0] = Ev3Opcode.OPINPUT_DEVICE_LIST;
             byteCommands[1] = Ev3Value.NUM8; // 1 byte to follow
-            byteCommands[2] = 33; // 0x21 ARRAY // TODO: ????
-            byteCommands[3] = 96; // 0x60 CHANGED // TODO: ????
-            byteCommands[4] = 225; // 0xE1 size of global var - 1 byte to follow // TODO: ????
-            byteCommands[5] = 32; // 0x20 global var index "0" 0b00100000 // TODO: ????
+            byteCommands[2] = 33; // 0x21 ARRAY // TODO: document
```

Comment:
I wanted to put in some reminders for someone to document some remaining EV3 hardware protocol things that didn't make it in the previous refactor.

Reply:
https://github.com/LLK/scratch-vm/issues/1568